subprojects/cli/src/main/java/org/gradle/cli/CommandLineOption.java

```diff
@@ -37,6 +37,7 @@ public Set<String> getOptions() {
         return options;
     }
 
+    @SuppressWarnings("UnusedReturnValue")
```

Comment:
❌ Please remove this

Reply:
@SuppressWarnings("UnusedReturnValue") has been removed from this PR,thanks